test/unit/fonts_spec.js

```diff
@@ -0,0 +1,13 @@
+/* globals describe, it, expect, beforeAll, afterAll,
+           checkProblematicCharRanges */
+
+'use strict';
+
+describe('Fonts', function() {
+  it('checkProblematicCharRanges', function() {
+    var EXPECTED_PERCENTAGE = 45;
```

Comment:
It comes from the fact that we currently utilize approximately `44%` of Private Use Area (PUA).
Currently we have no way of knowing how much of the PUA that the remapped characters can potentially make use of, and by having this test fail when you add a number of new char ranges it becomes a lot clearer how the PUA usage increases over time.

We obviously want to take care that we don't try to add more char ranges than the PUA can hold.
When you add new char ranges, you might have to increase this constant, which I feel should help both the developer and reviewer to notice that we're now using more of the PUA than before.

So by having this unit-test that will fail if you add a bunch of new char ranges, we will get advance warning if in the future we're about to run out of space in the PUA.

A bit long-winded perhaps, but I hope it makes sense! It not, just ask :-)